deploy-as-code/helm/charts/health-services/plan-service/values.yaml

```diff
@@ -0,0 +1,86 @@
+# Common Labels
+labels:
+  app: "plan-service"
+  group: "health"
+
+namespace: health
+
+# Ingress Configs
+ingress:
+  namespace: egov
+  enabled: true
+  zuul: true
+  context: "plan-service"
+
+# Init Containers Configs
+initContainers:
+  dbMigration:
+    enabled: true
+    schemaTable: "plan_schema"
+    image:
+      repository: "plan-service-db"
+
+# Container Configs
+image:
+  repository: "plan-service"
+replicas: "1"
+healthChecks:
+  enabled: true
+  livenessProbePath: "/plan-service/health"
+  readinessProbePath: "/plan-service/health"
+appType: "java-spring"
+tracing-enabled: true
+heap: "-Xmx192m -Xms192m"
+java-args: ""
+egov-mdms-service-host-key: "egov-mdms-service"
+
+# Additional Container Envs
+env: |
+  - name: SPRING_KAFKA_CONSUMER_GROUP_ID
+    value: plan-service
+  - name: SPRING_KAFKA_PRODUCER_KEY_SERIALIZER
+    value: org.apache.kafka.common.serialization.StringSerializer
+  {{- if index .Values "global" "tracer-errors-provideexceptionindetails" }}
+  - name: TRACER_ERRORS_PROVIDEEXCEPTIONINDETAILS
```

Comment:
what does this value correspond to?

Reply:
its to decide whether to print the whole stack trace or not during exception handling.